index.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const express = require('express');
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(morgan('dev'));
+app.use(express.static(path.join(__dirname, 'public')))
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.get('/', (request, response) => {
+  response.redirect('./clucks');
```

Comment:
Good thinking using a `redirect` to your `clucks/index` route

however, it is better to not include the `.` since you are redirecting to a path, not a file.
A slightly better way or writing this would be:

```javascript
response.redirect('/clucks');
```
without the `.`